src/tests/ui/planner/workitem-quickpreview.ts

```diff
@@ -141,6 +142,7 @@ export class WorkItemQuickPreview extends ui.BaseElement {
   async addCommentAndSave(comment: string) {
     await this.addComment(comment);
     await this.commentSaveButton.clickWhenReady();
+    await this.commentsBox.untilCount(3); 
```

Comment:
@Raunak1203 This might not work in all cases. What if I wish to add a comment on a workitem that has `0` comments? The `untilCount(3)` will wait indefinitely and lead to test failure

Reply:
done